include/nlohmann/detail/input/lexer.hpp

```diff
@@ -131,7 +131,7 @@ class lexer : public lexer_base<BasicJsonType>
     JSON_HEDLEY_PURE
     static char get_decimal_point() noexcept
     {
-        const auto loc = localeconv();
+        const auto* loc = localeconv();
```

Comment:
Technically you lost the local-`const` here, but meh, it doesn't really matter.

Reply:
You mean it should be `const auto* const loc`?